build.gradle.kts

```diff
@@ -18,6 +18,8 @@ plugins {
     checkstyle
     jacoco
     id("com.rameshkp.openapi-merger-gradle-plugin") version "1.0.4"
+    id ("org.eclipse.dataspaceconnector.dependency-rules") apply(false)
+    id("com.autonomousapps.dependency-analysis") version "1.0.0-rc04" apply (false)
```

Comment:
rc05 required for exclude annotations

Reply:
rc04 is the versions required for working exclude annotations, but I will update the version anyways